instagram/src/components/PostsContainer/Post.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import CommentSection from '../CommentSection/CommentSectionContainer';
+import LikeSection from './LikeSection';
+import PostHeader from './PostHeader';
+
+import './Posts.css';
+
+class Post extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      likes: props.post.likes
+    };
+  }
+  incrementLike = () => {
+    let likes = this.state.likes + 1;
+    this.setState({ likes });
```

Comment:
One of many approaches of incrementing and decrementing a like is by
Including an initial state `isLiked: false`

```
incrementLike = () => {
    if (!this.state.isLiked) {
      this.setState({likes: this.state.likes + 1, isLiked: true});
    } else {
      this.setState({likes: this.state.likes - 1, isLiked: false});
    }
  };
```